scripts/examples/s2_ard_non_prod.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#PBS -P u46
+#PBS -W umask=017
+#PBS -q normal
+#PBS -l walltime=2:30:00,mem=15GB,other=pernodejobfs
+#PBS -l wd
+#PBS -l storage=gdata/v10+scratch/v10+gdata/if87+gdata/fj7+scratch/fj7+scratch/u46+gdata/u46
+#PBS -l ncpus=1
+
+module use /g/data/v10/public/modules/modulefiles
+module use /g/data/v10/private/modules/modulefiles
+
+module load ard-scene-select-py3-dea/20230330
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SCRATCH=${DIR}/scratch
+mkdir -p $SCRATCH
+
+PKDIR=$SCRATCH/pkgdir$RANDOM
+mkdir -p $PKDIR
+
+ard-scene-select \
```

Comment:
i really like this separation of the params. Makes it easier to read and maintain.

Reply:
:)